content/docs/lists-and-keys.md

```diff
@@ -178,7 +178,7 @@ ReactDOM.render(
 ```javascript{2,3,9,10}
 function ListItem(props) {
   // Correct! There is no need to specify the key here:
-  return <li>{props.value}</li>;
+  return <li key={props.key}>{props.value}</li>;
```

Comment:
`key` is not reachable from the `props`. It is only handled by React. It's [in the docs](https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys) and even has a specific comment in the code you're changing. 

Does this clear things up?

Reply:
Yes, Thank you :)